### PR TITLE
Algolia docs

### PR DESCRIPTION
**Pull request by lenadoc**
## PR Description

TBD

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/pbc/all/search/algolia.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: Algolia
+description: Algolia is a search engine that
```
Comment (jeremyfourna) [inline docs/pbc/all/search/algolia.md:3]: For the description, we will use Algolia directly
> Algolia empowers Builders with Search and Recommendation services to create world-class digital experiences.

File: docs/pbc/all/search/algolia.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+---
+title: Algolia
+description: Algolia is a search engine that 
+---
+
+By default, all Spryker stores have [Elasticsearch](https://www.elastic.co/elasticsearch/) as their search engine. However, you can replace it with the Algolia one.
```
Comment (jeremyfourna) [inline docs/pbc/all/search/algolia.md:6]: I would rephrase it as 

Out of the box, Spryker is shipped with [Elasticsearch](https://www.elastic.co/elasticsearch/) as the default search engine. However, you can replace it with Algolia.

File: docs/pbc/all/search/algolia.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+The Algolia search engine that stands out from others due to its performance. With the Algolia app, your users can conduct advanced search of active concrete products in your store.
```
Comment (jeremyfourna) [inline docs/pbc/all/search/algolia.md:8]: I would remove this sentence, it does not help.

File: docs/pbc/all/search/algolia.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
+---
+title: Algolia
+description: Algolia is a search engine that 
+---
+
+By default, all Spryker stores have [Elasticsearch](https://www.elastic.co/elasticsearch/) as their search engine. However, you can replace it with the Algolia one.
+
+The Algolia search engine that stands out from others due to its performance. With the Algolia app, your users can conduct advanced search of active concrete products in your store. 
+
+{% info_block infoBox "Product offers and product reviews" %}
```
Comment (jeremyfourna) [inline docs/pbc/all/search/algolia.md:10]: This section is not true, in my opinion. We only send concrete product information; no product offers are sent there.
@matweew can you confirm?

File: docs/pbc/all/search/algolia.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+- Keywords
+- Currency
+- Gross Price
+- Net Price
+- Product Abstract SKU
+- Rating (average approved rating for a respective abstract product)
+- Images
+- Categories
+- Attributes
+- Merchants
```
Comment (jeremyfourna) [inline docs/pbc/all/search/algolia.md:34]: Marketplace data structure is not supported, so I would prefer to remove this line.

File: docs/pbc/all/search/disconnect-algolia.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+---
+title: Disconnect Algolia
+description: Find out how you can disconnect Algolia from your Spryker shop
+template: howto-guide-template
+---
+Disconnecting the Algolia app immediately and permanently deletes all the existing Algolia [index](/docs/pbc/all/search/algolia.html#indexes) and restores the default [Elasticsearch](https://www.elastic.co/elasticsearch/) search engine for your store.
+
+To disconnect the Algolia app from your store, do the following
```
Comment (jeremyfourna) [inline docs/pbc/all/search/disconnect-algolia.md:8]: Can we do a small screen recording here? That will be nicer from the UX side.

File: docs/pbc/all/search/integrate-algolia.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
+---
+title: Integrate Algolia
+description: Integrate Algolia in your Spryker shop
+template: howto-guide-template
+---
+
+To start using Algolia for your shop, you need an account with Algolia. You can create the account at the [Algolia website](https://www.algolia.com).
+
+To integrate the Algolia app, do the following:
```
Comment (jeremyfourna) [inline docs/pbc/all/search/integrate-algolia.md:9]: We should also add a small recording of all those steps

File: docs/pbc/all/search/integrate-algolia.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+8. In the top right corner of the Algolia app details page, click **Configure**.
+9. In the **Configure** pane, populate the _APPLICATION ID_, _SEARCH-ONLY API KEY_, and _ADMIN API KEY_ fields with the values from step 6.
+![algolia-settings](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/search/algolia/integrate-algolia/algolia-settings.png)
+10. Click **Save**.
+
+The Algolia app is now integrated to your store and starts exporting your product data automatically. Wait for a few minutes and go back to the Algolia website, to the **Overview -> Search -> Index** page. 
+
+{% info_block infoBox "Info" %}
+
+The more products you have, the longer you have to wait until Algolia finishes the export.
```
Comment (jeremyfourna) [inline docs/pbc/all/search/integrate-algolia.md:31]: @matweew do you know how much time is taken on average for xxx products?

**Review by jeremyfourna — CHANGES_REQUESTED** [review]

**Review by jeremyfourna — DISMISSED** [review]
OK for me except a small comment.

File: docs/pbc/all/search/algolia.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+description: Algolia empowers Builders with Search and Recommendation services to create world-class digital experiences.
+---
+
+Spryker is shipped with [Elasticsearch]((https://www.elastic.co/elasticsearch/)) as the default search engine. However, you can replace it with [Algolia](https://www.algolia.com/).
+
+The Algolia search engine stands out from others due to its performance. With the Algolia app, your users can conduct advanced search of active concrete products in your store. 
+
+{% info_block infoBox "Product reviews" %}
+
+Algolia searches in product reviews as well. However, product reviews are represented in product attributes in the Algolia search results.
```
[the remark this answers is not in the document]
Comment (matweew) [reply docs/pbc/all/search/algolia.md:12]: @lenadoc 
If you mean to search by content (text) of product reviews - it's not working.
if you mean the ability to filter products by average rating (only 5 stars or between 3.5 and 4.5) - it will work
